build.sbt

```diff
@@ -24,6 +25,9 @@ commands := commands.value.filterNot { command =>
   }
 }
 
+ThisBuild / reproducibleBuildsCheckResolver :=
+  "Apache Pekko Staging".at("https://repository.apache.org/content/groups/staging/")
```

Comment:
I understand that this useful but users might want to verify against maven central or some other repo. Would it be possible to add a comment about other possible repo values - possibly a link to some centralised doc.

Reply:
I was planning to document this in our [release process](https://github.com/apache/incubator-pekko-site/wiki/Pekko-Release-Process). This makes sense as a default because almost all of the time the use case is to verify against jars in staging, if something has already been released and its not in staging then something went wrong.